src/app/components/fees-row/fees-row.tsx

```diff
@@ -34,6 +37,12 @@ export function FeesRow(props: FeeRowProps): React.JSX.Element {
   const isCustom = feeTypeField.value === FeeTypes[FeeTypes.Custom];
   const selectedItem = Number(FeeTypes[feeTypeField.value]);
 
+  useEffect(() => {
+    if (props.defaultFeeValue) {
+      setTimeout(() => handleSelectFeeEstimateOrCustomField(FeeTypes.Custom));
+    }
+  }, [props.defaultFeeValue])
```

Comment:
This data flow of having an effect inside the FeeRow component set a fee, even though it's disabled, is hard to follow. 

Any other ways we can do this?